src/mp/proxy.cpp

```diff
@@ -183,11 +183,11 @@ void EventLoop::addAsyncCleanup(std::function<void()> fn)
     startAsyncThread();
 }
 
-EventLoop::EventLoop(const char* exe_name, LogFn log_fn, void* context)
+EventLoop::EventLoop(const char* exe_name, LogOptions log_opts, void* context)
     : m_exe_name(exe_name),
       m_io_context(kj::setupAsyncIo()),
       m_task_set(new kj::TaskSet(m_error_handler)),
-      m_log_fn(std::move(log_fn)),
+      m_log_opts(log_opts),
```

Comment:
In commit "event loop: make log options configurable" (e6013d793e1be2d1becdbe422c7c4ce5bc1d9e28)

I think it might be a little better if the PR avoided changing the EventLoop signature, and just assigned `m_log_opts.log_fn = log_fn` in the body of the constructor. This would prevent downstream code in bitcoin core that uses the EventLoop from breaking, and make it a little easier to merge the subtree PR, not breaking the build in the subtree bump commit. Another option to stay compatible could be to have two constructors, though I like that idea a little less.

Reply:
Thanks for review
Done.